src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4TokensPanel/hooks/useV4ReservedTokensSubPanel.ts

```diff
@@ -0,0 +1,81 @@
+import { WeiPerEther } from '@ethersproject/constants'
+import { ONE_BILLION } from 'constants/numbers'
+import { useJBContractContext, useJBRulesetMetadata, useReadJbTokensTotalCreditSupplyOf } from 'juice-sdk-react'
+import useProjectOwnerOf from 'packages/v4/hooks/useV4ProjectOwnerOf'
+import { useV4ReservedSplits } from 'packages/v4/hooks/useV4ReservedSplits'
+import { formatV4SplitPercent } from 'packages/v4/utils/math'
```

Comment:
Can we pls remove all uses of this and replace it with `SplitPortion` from the SDK?

`new SplitPortion(value).format()`

Reply:
Done